video/out/opengl/video.c

```diff
@@ -1711,6 +1742,11 @@ static void user_hook(struct gl_video *p, struct img_tex tex,
     float w = eval_szexpr(p, tex, shader->width);
     float h = eval_szexpr(p, tex, shader->height);
 
+    // Make sure we at least create a legal FBO on failure, since it's better
+    // to do this and display an error message than just crash OpenGL
+    w = isnan(w) ? 1.0 : w;
+    h = isnan(h) ? 1.0 : h;
```

Comment:
Maybe even isnormal(), considering there's no valid use case for denormalized texture sizes.